ArduSub/Sub.h

```diff
@@ -423,7 +423,7 @@ class Sub : public AP_HAL::HAL::Callbacks {
 #endif
 
 #if AVOIDANCE_ENABLED == ENABLED
-    AC_Avoid avoid{ahrs, inertial_nav, fence, g2.proximity, &g2.beacon};
+    AC_Avoid avoid{ahrs, fence, g2.proximity};
```

Comment:
hummm beacon is still needed !

Reply:
Gah.  That's the *second* time that's caught me!  Optional parameters suck.  Fixed.